docs/concepts/resolution.md

```diff
@@ -296,6 +296,18 @@ To ensure reproducibility, messages for unsatisfiable resolutions will not menti
 distributions were excluded due to the `--exclude-newer` flag — newer distributions will be treated
 as if they do not exist.
 
+## Source distribution
+
+Most packages publish their source distributions as gzip tarball (`.tar.gz`) or zip (`.zip`)
```

Comment:
We should clarify that PEP 625 has decided that `.tar.gz` is the canonical source distribution filename, all other extensions should be considered legacy or experimental.